wsgi/wsgi_app.py

```diff
@@ -174,9 +175,18 @@ def is_data_suspicious(event_list):
     the actual birth of the universe and the reweaving of past fates into
     current events has come to hunt us and our beloved data.
 
+    Apr 7, 2019 0:00 is the default time after a cold start without GPS signal.
+    The DAQ will happily send events even when no GPS signal has been
+    acquired (yet). Events with timestamp Apr 7, 2019 are most probably caused
+    by no or bad GPS signal. Such events must be eigenstates of suspiciousness.
+
     """
     for event in event_list:
         if event['header']['datetime'].year < 2013:
```

Comment:
I think we could also bump this value (2013) to something more recent.. But for now I agree from now on block events from April 7th.